pyvrp/_Solution.pyi

```diff
@@ -107,6 +114,17 @@ class Solution:
             A list of ``(pred, succ)`` tuples that encode for each client their
             predecessor and successors in this solutions's routes.
         """
+    def get_assigned_vehicle_types(self) -> List[int]:
```

Comment:
It's not strictly necessary, just like binding `get_neighbours` to Python is not 'necessary'/not currently used except for testing. I do think it's good to support it though, it can e.g. be used to define a custom distance function in Python.